lib/ts/recipe/multitenancy/utils.ts

```diff
@@ -15,18 +15,7 @@ export function normaliseMultitenancyConfig(config?: UserInput): NormalisedConfi
 
```

Comment:
not related to this file. but as part of the final review, we should go through all the Ui on storybook once together. So call me when you see this please.